### ISSUE TITLE
.NET 6 projects aren't created properly

### ISSUE THREAD

**Issue by mikael-bergstrom-ntisthlm** [rsaz/csharp-snippet-productivity#9]
The Create Project dialog successfully detects .NET 6 as an available framework, but it seems the commands to create projects don't work.

Here's what the log tells me C# Toolbox is trying to do when I tell it I want a .NET 6 console application:

`dotnet new console --language c# -n Test6 -o 'd:\Test6\Test6' -f  --force`

And this is the corresponding line run when creating a .NET 5 project:

`dotnet new console --language c# -n Test5 -o 'd:\Test5\Test5' -f net5.0 --force`

In short, it seems like the "net6.0" option isn't included.

**Comment by rsaz** [comment]
Thank you for highlighting this problem. Now the extension have support to net core 6.0
